corehq/apps/userreports/tests/test_transforms.py

```diff
@@ -21,6 +21,34 @@ def testInvalidType(self):
             })
 
 
+class NumberFormatTransformTest(SimpleTestCase):
+
+    def setUp(self):
+        self.transform = TransformFactory.get_transform({
+            "type": "number_format",
+            "format_string": "{0:.0f}"
+        }).get_transform_function()
+
+    def test_int(self):
+        self.assertEqual('11', self.transform(11))
+
+    def test_decimal(self):
+        self.assertEqual('11', self.transform(11.23))
+
+    def test_strings(self):
+        tests = (
+            ('11', '11'),
+            ('11.23', '11'),
+            ('notanumber', 'notanumber'),
+            ('', ''),
+        )
```

Comment:
not 100% sure i like this better, though looks like a useful utility and was worth the time it took to learn how to work with it. https://github.com/dimagi/commcare-hq/pull/9930